app/controllers/SessionsController.scala

```diff
@@ -62,6 +70,15 @@ class SessionsController @Inject()(messagesApi: MessagesApi,
                                   ) extends KnolxAbstractController(controllerComponents) with I18nSupport {
 
   val usersRepo: UsersRepository = usersRepository
+  implicit val KnolxSessionInfoFormat: OFormat[KnolxSession] = Json.format[KnolxSession]
+  implicit val SessionSearchResultInfoFormat: OFormat[SessionSearchResult] = Json.format[SessionSearchResult]
+
+  val sessionSearchForm = Form(
+    mapping(
+      "email" -> optional(nonEmptyText),
+      "page" -> number.verifying("Invalid feedback form expiration days selected", number => number >= 1)

```

Comment:
This
number => number >= 1
can also be written like _ >= 1
just a suggestion for being consistent